Documentation/README.md

```diff
@@ -12,6 +12,6 @@ Once you have a Rook cluster running, you can use the `rook` tool to create and
 - [Using Rook Guide](client.md)
 
 
-## Troubleshooting Toolbox
+## Advanced Configuration and Troubleshooting
```

Comment:
we have this same paragraph and link from the main readme at the root of the repo too.  should we be making a similar edit to the main readme too?  or somehow remove the duplication?